libs/blocks/src/lib/fast-payment/base/index.tsx

```diff
@@ -10,34 +10,46 @@ const Base: React.FC<FastPaymentProps> = ({
   description,
   link,
   disclaimer,
-  content: { cards = [], cols = 'three', dense },
+  content: { cards = [], cols = 'three', dense, sliderClass },
+  isRtl = false,
+  className,
 }) => {
   return (
-    <FluidContainer className="bg-background-primary-container py-general-4xl">
-      <div className="flex flex-col items-center text-center">
-        <Heading.H2>{title}</Heading.H2>
-        {description && <Text className="pt-general-lg">{description}</Text>}
+    <FluidContainer
+      className={qtJoin(
+        'flex flex-col items-center py-general-4xl text-center',
+        className,
+      )}
+      dir={isRtl ? 'rtl' : 'ltr'}
+      id="fast-payment-container"
```

Comment:
please use `data-testid` instead of `id`.

Reply:
changed `id` to `data-testid`